mne/decoding/search_light.py

```diff
@@ -0,0 +1,491 @@
+            Predicted values for each estimator.
+        """
+        return self._transform(X, 'predict')
+
+    def predict_proba(self, X):
+        """Estimate probabilistic estimates of each data slice with all
+        possible estimators.
+
+        Parameters
+        ----------
+        X : array, shape (n_samples, n_features, n_estimators)
+            The training input samples. For each data slice, a clone estimator
+            is fitted independently.
+
+        Returns
+        -------
+        y_pred : array, shape (n_samples, n_estimators, n_slices, n_classes)
+            Predicted values for each estimator.
+
+        Notes
+        -----
+        This requires base_estimator to have a `predict_proba` method.
+        """
+        return self._transform(X, 'predict_proba')
+
+    def decision_function(self, X):
+        """Estimate distances of each data slice to all hyperplanes.
+
+        Parameters
+        ----------
+        X : array, shape (n_samples, n_features, n_estimators)
+            The training input samples. Each estimator output the distance to
+            its hyperplane: e.g. [estimators[ii].decision_function(X[..., ii])
+                                  for ii in range(n_estimators)]
+
+        Returns
+        -------
+        y_pred : array, shape (n_samples, n_estimators, n_slices, n_classes)
+            Predicted values for each estimator.
+
+        Notes
+        -----
+        This requires base_estimator to have a `decision_function` method.
+        """
+        return self._transform(X, 'decision_function')
+
+    def score(self, X, y):
+        """Returns the score obtained for each combination of estimators and
+        tested dimensions.
+
+        Parameters
+        ----------
+        X : array, shape (n_samples, n_features, n_estimators)
+            The input samples. For each data slice, the corresponding estimator
+            score the prediction:
+            e.g. [estimators[ii].score(X[..., ii], y)
+                  for ii in range(n_estimators)]
+        y : array, shape (n_samples,)
+            The target values.
+
+        Returns
+        -------
+        score : array, shape (n_samples, n_estimators, n_slices, n_score_dim)  # noqa
+            Score for each estimator / data slice couple.
+
+        Notes
+        -----
+        This requires base_estimator to have a `decision_function` method.
+        """
+        self._check_Xy(X)
+        # For predictions/transforms the parallelization is across the data and
+        # not across the estimators to avoid memory load.
+        parallel, p_func, n_jobs = parallel_func(_gl_score, self.n_jobs)
+        X_splits = np.array_split(X, n_jobs, axis=-1)
+        score = parallel(p_func(self.estimators_, x, y) for x in X_splits)
+
+        if n_jobs > 1:
+            score = np.concatenate(score, axis=1)
+        else:
+            score = score[0]
+        return score
+
+
+def _gl_transform(estimators, X, method):
```

Comment:
ok I see, ... it's both in some sense. The content remains a prediction. If you want to keep transform add at least a small doc comment or so that links it to predictions and explains this point.